.github/actions/docker-run/action.yml

```diff
@@ -80,7 +80,7 @@ runs:
           ${{ inputs.device }}
           -w ${{ github.workspace }}
         run: |
-          if [ ${{ inputs.install_wheel }} ]; then
+          if [ ${{ inputs.install_wheel }} = "true" ]; then
```

Comment:
Need to use as a string comparison - the value is not actually boolean. There is an existing Github issue:
- https://github.com/actions/runner/issues/1483

Reply:
does it matter if its "" or ''?
I didn't realize this was a thing so maybe we should double check if we have other comparisons here too

Reply:
I think both "" and '' evaluate to a string in bash so it does not matter.
Do you think '' is more consistent?

Reply:
i think as long as we stick to one convention it is fine.